android/src/main/java/io/sentry/react/RNSentryModuleImpl.java

```diff
@@ -68,6 +72,8 @@ public class RNSentryModuleImpl {
 
     public static final String NAME = "RNSentry";
 
+    private static JavaScriptExecutorFactory javaScriptExecutorFactory = null;
```

Comment:
L: is there a way to not make it static?

Reply:
I need to update this code, this was from the first try when I thought we will need the `javaScriptExecutorFactory` to use it's start and stop profiling methods.